Description: disable version checks on startup
 Rapid-photo-downloader performs a check for new versions on startup
 upstream. In Debian, users cannot perform this update themselves
 through the mechanisms offered upstream: they must perform updates
 through the Debian packaging system. This check will, when the
 package trickle down in Debian, trigger needless warnings when
 upstream performs new releases that are not backported to stable or
 LTS releases. 
Origin: upstream, https://discuss.pixls.us/t/rapid-photo-downloader-0-9-9-is-released/6912
Forwarded: not-needed
Last-update: 2018-03-14
Author: Antoine Beaupre <anarcat@debian.org>

--- a/raphodo/constants.py
+++ b/raphodo/constants.py
@@ -31,7 +31,7 @@
 
 # If set to True, the ability to check for a new version will be removed
 # from the user interface and disabled in program logic.
-disable_version_check = False
+disable_version_check = True
 
 
 class CheckNewVersionDialogResult(IntEnum):
